spec/rails_helper.rb

```diff
@@ -8,7 +8,7 @@
 RSpec.configure do |config|
   config.fixture_path = "#{::Rails.root}/spec/fixtures"
 
-  config.use_transactional_fixtures = true
+  config.use_transactional_fixtures = false
```

Comment:
@mikedao: I decided to include the database cleaner gem yesterday because some things weren't getting removed from the DB between tests. Per [RSpec docs](https://relishapp.com/rspec/rspec-rails/docs/transactions), setting this to false always the outside gem to take over cleaning from the built-in RSpec methods:
> If you prefer to manage the data yourself, or using another tool like database_cleaner to do it for you, simply tell RSpec to tell Rails not to manage transactions: